great_expectations/expectations/metrics/column_aggregate_metrics/column_partition.py

```diff
@@ -90,6 +90,12 @@ def _get_evaluation_dependencies(
                 "column.max": MetricConfiguration(
                     "column.max", metric.metric_domain_kwargs
                 ),
+                "table.columns": MetricConfiguration(
+                    metric_name="table.columns",
+                    metric_domain_kwargs=metric.metric_domain_kwargs,
+                    metric_value_kwargs=None,
+                    metric_dependencies=None,
+                ),
```

Comment:
Whenever we're instantiating a new MetricConfiguration for "table.columns" and passing it `metric.metric_domain_kwargs`, don't we need to pop `"column"` from metric.metric_domain_kwargs` if it's present (like you did above)? Or are we just certain that metric_domain_kwargs here will never have a "column" key?

Reply:
Yes!  I did it in another commit.  When this is ready for re-review, please double check that it is there.